storage/filesystem/module.go

```diff
@@ -0,0 +1,14 @@
+package filesystem
+
+import (
+	"srcd.works/go-git.v4/storage"
+	"srcd.works/go-git.v4/storage/filesystem/internal/dotgit"
+)
+
+type ModuleStorage struct {
```

Comment:
Do we really need this type?, can we just make dotgit implement `storage.ModuleStorer` instead.